firefox-ios/Client/Frontend/TrackingProtection/TrackigProtectionRedux/TrackingProtectionState.swift

```diff
@@ -134,11 +201,24 @@ struct TrackingProtectionState: StateType, Equatable, ScreenState {
                 trackingProtectionEnabled: state.trackingProtectionEnabled,
                 connectionSecure: state.connectionSecure,
                 showDetails: false,
-                showBlockedTrackers: false
+                showBlockedTrackers: false,
+                showsClearCookiesAlert: false,
+                shouldClearCookies: false,
+                shouldUpdateBlockedTrackerStats: false
             )
         default:
-            return state
+            return TrackingProtectionState(
+                windowUUID: state.windowUUID,
+                shouldDismiss: false,
+                showTrackingProtectionSettings: false,
+                trackingProtectionEnabled: state.trackingProtectionEnabled,
+                connectionSecure: state.connectionSecure,
+                showDetails: false,
+                showBlockedTrackers: false,
+                showsClearCookiesAlert: false,
```

Comment:
nit: instead of all the booleans, can follow similar to how we show certain views using `DisplayType` in `BrowserViewControllerState`?

Reply:
yes it would be much cleaner that way, but I wanted to do the final navigation cleanup on a different PR to avoid making this one too big (it's already quite big) and to better organise the coordinator on the iPhone/iPad differences

Reply:
sounds good, thank you!